src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
```

Comment:
The class should not begin with an empty line. Please remove the empty line to follow the coding conventions.